storm-core/src/clj/backtype/storm/cluster.clj

```diff
@@ -248,9 +256,9 @@
   [id]
   (str ASSIGNMENTS-SUBTREE "/" id))
 
-(defn code-distributor-path
-  [id]
-  (str CODE-DISTRIBUTOR-SUBTREE "/" id))
+(defn blobstore-path
+  [key]
+  (str BLOBSTORE-SUBTREE "/" key))
```

Comment:
There is a server side validation for this, im adding the client side validation just to reinforce it